cmd/validate_editorconfig.go

```
@@ -5,6 +5,7 @@
 	"os"
 	"strings"
 
+	"github.com/charmbracelet/log"
```

golangci-lint (1.64.8)

[error] import "github.com/charmbracelet/log" imported without alias but must be with alias "log" according to config

(importas)